templates/client/index.html.twig

```diff
@@ -37,7 +39,43 @@
 				</div>
 			</div>
 		</div>
-		<h2 class="text-center my-5">Votre objectif du mois : {{ client.monthName}}</h2>
-
+		<h2 class="text-center my-5">Votre objectif du mois :
+			{{ client.monthName}}
+		</h2>
+		<ul class="container nav nav-pills mb-3" id="pills-tab" role="tablist">
+			{% for session in client.getSessions() %}
```

Comment:
non, `in client.sessions`, coté twig, il appelle implicitement le getter correspondant s'il existe